.github/workflows/review-bot.yml

```diff
@@ -26,7 +26,7 @@ jobs:
           private_key: ${{ secrets.TEAM_APP_KEY }}
       # !This must always point to main. 
       # Change it for the PRs but remember to change it back
-      - uses: paritytech/review-bot@main
+      - uses: paritytech/review-bot@and-rule
```

Comment:
leftover?

Reply:
Intended for now so you can see how it works. Will fix this when the approvals have been met.